doc-site/components/imports.tsx

```diff
@@ -2,30 +2,33 @@ import { css } from '@emotion/react';
 import { tokens } from '@datacamp/waffles/tokens';
 import { ErrorBoundary } from '@datacamp/waffles/error-boundary';
 
-import formattedModuleExports from '../helpers/formatted-module-exports';
+import generateExports from '../helpers/formatted-module-exports';
 
 import basicTheme from './editor-theme';
 import Highlight from './editor-highlight';
 import CodePreview from './code-preview';
 
 const previewStyle = css`
   border-radius: ${tokens.borderRadius.medium};
+  margin-bottom: ${tokens.spacing.medium};
 `;
 
 type ImportsProps = {
   module: Record<string, unknown> | string[];
   path: string;
+  additionalTypes?: string[]; // Used when the types cannot be programmatically determined from exported components (e.g. Toast)
```

Comment:
Very minor, but I prefer 
```suggestion
  /** Used when the types cannot be programmatically determined from exported components (e.g. `Toast`) */
  additionalTypes?: string[];
```
so that your IDE will pick it up (e.g. VSCode will show it in a tooltip when you hover the type elsewhere)